ctapipe/reco/hillas_intersection.py

```diff
@@ -309,7 +310,7 @@ def reconstruct_nominal(self, hillas_parameters):
 
         """
         if len(hillas_parameters) < 2:
-            return None  # Throw away events with < 2 images
+            return (np.nan, np.nan, np.nan, np.nan)  # Throw away events with < 2 images
```

Comment:
I see changes in the return values here but no changes that accept these changes...

What is the effect of changing the simple `None` to a tuple of 4 `np.nan`?

Reply:
Looking at the unit test, I think the effect is to strictly increase the correctness of the code. Or more plainly: no code could actually handle the `None` in the first place, the two places where `reconstruct_nominal` is actually called both expect and indexable object :)

Reply:
yes, sorry I should have commented here: returning None there was actually  a bug that i encountered. The useage expects a 4-tuple, and the code would have just crashed if None was returned, so I fixed it (it's not strictly part of this PR though)